.gitignore

```diff
@@ -114,3 +114,5 @@ env/
 
 ### Configuration files ###
 .env
+.venv
+geckodriver.log
```

Comment:
Good additions, adding the geckodriver binary might also be worthwhile as it's quite large, and users on different operating systems may need to download a separate one anyway